config.json

```diff
@@ -20,6 +20,12 @@
             "Class" : "DNWS.DelayPlugin",
             "Preprocessing" : "false",
             "Postprocessing" :"false" 
+        },
+        {
+            "Path" : "clientinfo",
+            "Class" : "DNWS.Clientinformation",
```

Comment:
1. Try to follow the pattern, we put suffix "Plugin", so the other can know that this is a plugin
2. You should follow the style guideline, i.e., you should use title case, like "ClientInformationPlugin" for any class name.